### ISSUE TITLE
ERR_TOO_MANY_REDIRECTS

### ISSUE THREAD

**Issue by nicktg0** [engintron/engintron#1165]
Hi,

some months ago i had installed the engintron on the server and it worked pretty well. After a little time period i wanted to try a different cache plugin (unixy varnish). So, i had remove the engintron according to the documentation and i had install the varnish plugin and everything is had fine.
One month later I decided I didn't want to use the varnish plugin and I wanted to go back to the engintron. I had remove the varnish plugin and i had install the engintron plugin again. 

The  installation completed successfully and engintron is enabled but the sites with ssl has going down. All sites with ssl return this error: ERR_TOO_MANY_REDIRECTS (The sites without ssl works fine). If I disable the engintron they come back online.

I think the problem is related with the varnish plugin. Any idea about how I can solve it?

I use cPanel v86.0.16, centos 7.7, php-pfm 7.1

**Comment by fevangelou** [comment]
Perhaps you didn't properly remove the Varnish plugin or if there's some uninstaller, it's not properly uninstalling from the system.

In any case, Nginx would never redirect between HTTP <-> HTTPS.

Check if there are any leftover rules in .htaccess files from proxying HTTPS traffic to HTTP as Varnish does not support HTTPS (in the "free" edition) and as such it requires proxying from HTTPS (possibly Apache) to Varnish (HTTP) which in turn fetches content from Apache (HTTP - not port 80) again.

Varnish is for aggressive caching for media sites (in most cases). Anything that requires user interaction (signups, carts, user generared content like forums etc.) requires a unique configuration in Varnish. There is no one-size-fits-all, unlike Engintron that is... ;)

**Comment by nicktg0** [comment]
All .htaccess files is ok without rules from varnish plugin. 

The varnish plugin may not have been removed successfully but the uninstallation was based on the documentation.
When first time install it there was a similar problem. The solution from support was this command: python /scripts/unixyvarnish/process_varnish_ssl --force

I tried to install the engintron again and again but the result remains the same.
